great-idea/css/services.css

```diff
@@ -0,0 +1,296 @@
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+.container {
+    width: 880px;
+    margin: 0 auto;
+}
+
+.navigation {
+    display: flex;
+    justify-content: space-between;
+    font-size: 20px;
+    margin: 20px 0;
+}
+
+.navigation a {
+    color: gray;
+    text-decoration: none;
+}
+
+.services-header-img {
+    margin-bottom: 20px;
+}
+
+.services {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: flex-start;
+    border-bottom: 1px solid black;
+    padding-bottom: 20px;
+}
+
+.services h3 {
+    font-size: 24px;
+    font-weight: bold;
+}
+
+.blocks {
+    display: flex;
+    flex-wrap: wrap;
+}
+
+.digital-design {
+    background: whitesmoke;
+    display: flex;
+    flex-wrap: wrap;
+    width: 420px;
+    align-content: flex-start;
+    border: 1px solid black;
+    margin-top: 25px;
+}
+
+.digital-design h3 {
+    font-size: 20px;
+    padding: 15px;
+}
+
+.digital-design p {
+    align-content: flex-start;
+    padding: 0 15px;
+}
+
+button {
+    justify-content: flex-start;
+    text-align: center;
+    font-size: 16px;
+    border-radius: 5px;
+    margin: 15px;
+    width: 150px;
+    height: 30px;
```

Comment:
same thing as index - width, height, and text-align could all be condensed into padding, and changing the flex-direction on the parent would save you the margin as well.